### DISCUSSION TITLE
Fcpxml not working with Davinci 19

### DISCUSSION THREAD

**Discussion by spacetravelerfox** [CommandPost/CommandPost#3374] [Technical Support] [answered]
Hello, I have problems Final Cut xml file not importing to Davinci. I read that command post can fix the file so it imports correctly. But somehow when I drag the project to command post and it fixes it, I import it to Davinci and get the error that Davinci only supports xml files up to version 1.10. Since I'm not fixing the actual xml file, I can't export version 1.10, so I'm assuming FCPX exports the newest version?

**Comment by randomeizer** [comment]
When exporting from FCP, Apple lets you select the most recent or second most recent FCPXML version (1.12 or 1.11 currently).

I'm a bit surprised that Resolve doesn't support 1.11 or 1.12. They usually keep up-to-date at least one version behind.

However, we do have an app which will translate between older versions, all the way back to FCPXML 1.6, called Capacitor:

https://capacitor.pro

That should help if Black Magic doesn't update the version the support.

**Comment by estudioek** [comment]
Yes, I just tried today with FCPX 18.1 and it does not work with Davinci 19 and there is no way to choose an older XML format... Hope you guys can fix this soon

**Reply by latenitefilms** [reply, marked as answer]
Just open it up in TextEdit and change the version at the top.
